### DISCUSSION TITLE
From Aspen Hysys to DWSIM

### DISCUSSION THREAD

**Discussion by ofvicente** [DanWBR/dwsim#531] [General]
Hi,

is it possible to translate "easily" a model of a process made in Aspen Hysys to DWSIM?

Thanks in advance.

**Comment by yoSimulation** [comment]
> Hi,
> 
> is it possible to translate "easily" a model of a process made in Aspen Hysys to DWSIM?
> 
> Thanks in advance.

Hi,

If you send me XML hysys model, I will try to convert your model for testing my model converter

**Reply by jamartinh** [reply]
Wow, are you planning to deploy this model converter ?

**Reply by yoSimulation** [reply]
We are currently undergoing closed beta testing

**Reply by jamartinh** [reply]
Can I be inside the beta club?

**Reply by yoSimulation** [reply]
you should fill [this form](https://forms.yandex.ru/u/654cce0f5d2a060015717f8c/)
